awx_collection/plugins/inventory/tower.py

```diff
@@ -98,15 +98,15 @@
 # ansible-inventory -i @tower_inventory --list
 '''
 
-import re
 import os
-import json
+import re
 from ansible.module_utils import six
-from ansible.module_utils.urls import Request, urllib_error, ConnectionError, socket, httplib
+from ansible.module_utils.urls import Request
 from ansible.module_utils._text import to_text, to_native
-from ansible.errors import AnsibleParserError, AnsibleOptionsError
+from ansible.errors import AnsibleOptionsError
 from ansible.plugins.inventory import BaseInventoryPlugin
 from ansible.config.manager import ensure_type
+from ..module_utils.ansible_tower import make_request
```

Comment:
Style nit pick, but in other places we _tend_ to put a blank line between imports of different types. Following that here would dictate a blank line after `import re` and then one before the module_utils import too.